oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/reactive/result/method/annotation/OAuth2AuthorizedClientResolver.java

```diff
@@ -70,6 +78,7 @@ public OAuth2AuthorizedClientResolver(
 	 * If true, a default {@link OAuth2AuthorizedClient} can be discovered from the current Authentication. It is
 	 * recommended to be cautious with this feature since all HTTP requests will receive the access token if it can be
 	 * resolved from the current Authentication.
+	 *
```

Comment:
Please remove any changes that are formatting related. Only code changes should be included.

Reply:
Same as above: I don't remember adding a blank line between the text description in the comment & the @param line, but I see I did...I've removed it.